keyboards/mikeneko65/rev4/readme.md

```diff
@@ -0,0 +1,17 @@
+# Mikeneko 65 V4
+
+A simple 65% keyboard
+
+* Keyboard Maintainer: [takishim](https://github.com/takishim)
+* Hardware Supported: Mikeneko 65 V4
+* Hardware Availability: [Open source on GitHub](https://github.com/takishim/mikeneko65/tree/v4)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```